testy/tagged.go

```diff
@@ -0,0 +1,96 @@
+package testy
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+const (
+	TagUnit        = "unit"
+	TagIntegration = "integration"
+	TagPostgres    = "postgres"
+	TagRabbit      = "rabbit"
+)
+
+// TaggedTestsEnvVar defines the name of the environment variable for the tagged tests.
+// Example:
+// 		TaggedTestsEnvVar="TEST_TAGS"
+//		env TEST_TAGS="unit" go test ./...
+var TaggedTestsEnvVar = "TEST_TAGS"
+
+// TaggedTest runs the test if the provided tag matches at least one runtime tag.
+// Example:
+// 		func TestSomething(t *testing.T) {
+//    		TaggedTest(t, "unit")
+//    		...
+// 		}
+// Run with:
+// 		env TEST_TAGS="unit,integration" go test ./...
+func TaggedTest(t *testing.T, testTag string) {
```

Comment:
the naming feels off to me, what do you think of these names instead?
- RequireTestTag()
- RequireAnyTestTags()
- RequireAllTestTags()

Reply:
or maybe RequireOneOfTestTags() instead of RequireAnyTestTags()